DisplayBundle/Manager/CacheableManager.php

```diff
@@ -61,13 +62,18 @@ protected function setResponseStatus(Response $response, $status)
      * 
      * @param Response $response
      * @param int      $maxAge
+     * @param bool     $hasEsi
      */
-    protected function setResponseMaxAge(Response $response, $maxAge)
+    protected function setResponseMaxAge(Response $response, $maxAge, $hasEsi = false)
```

Comment:
As setResponseCacheParameters always call setResponseMaxAge with $hasEsi, you should remove the default value here